dashboard/app/models/script.rb

```diff
@@ -342,6 +342,31 @@ def hoc_finish_url
     end
   end
 
+  def summarize
+    summary = {
+      id: id,
+      name: name,
+      stages: []
+    }
+    if trophies
+      summary[:trophies] = Concept.cached.map do |concept|
+        {
+          id: concept.name,
+          name: I18n.t("data.concept.description.#{concept.name}"),
+          bronze: Trophy::BRONZE_THRESHOLD,
+          silver: Trophy::SILVER_THRESHOLD,
+          gold: Trophy::GOLD_THRESHOLD
+        }
+      end
+    end
+
+    stages.select{|s| s.script_levels.to_a.count > 0}.sort_by(&:position).each do |stage|
```

Comment:
I was seeing this build break before df626f3fa06ee1e35f5ee9031536e6d00ab3cfd6:
```
FAIL ReportsControllerTest#test_should_have_one_game_group_if_one_stage (0.52s)
  Expected: 1
    Actual: 2
  test/controllers/reports_controller_test.rb:81:in `block in <class:ReportsControllerTest>'
```
https://travis-ci.org/code-dot-org/code-dot-org/builds/53453912

But the correct fix is probably to update the rake test, not add an extra condition in the code.